README.md

```diff
@@ -4,15 +4,24 @@ CLI for Node.js is an interactive tool that guides you through all the steps of
 
 ## Prerequisites
 The following software is required to work with SuiteCloud CLI for Node.js:
-- Node.js version 12.14 LTS
+- Node.js version 12.14.0 LTS or greater
 - Java JDK version 11
 
 Read the full list of prerequisites in [NetSuite's Help Center](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_1558708810.html).
+
+## Supported Versions
+Although you may have access to older versions of CLI for Node.js, only the versions in this section are officially supported.
+
+Each version of CLI for Node.js works optimally with a specific NetSuite version. To check the  equivalence among verisons, see the following table:
+
+| NPM | NetSuite |
```

Comment:
That seems a bit confusing to me. 
NPM version, or suitecloud cli version?
Npm itself has versions. https://nodejs.org/en/download/releases/

Reply:
What I mean to say here is the CLI for Node.js version in NPM, but not the NPM version users need to have installed to make it work. As I understood, we'll need a new version number to be hosted in NPM.